pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/creator/IndexCreationContext.java

```diff
@@ -59,6 +61,10 @@
 
   boolean hasDictionary();
 
+  Comparable getMinValue();
```

Comment:
Please don’t add raw types to this interface, Comparable<?> would be much better, otherwise the user needs to use raw types too